src/StickyPopup/StickyPopup.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Snackbar from '@material-ui/core/Snackbar'
+import SnackbarContent from '@material-ui/core/SnackbarContent'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+import StickyPopupDismiss from './StickyPopupDismiss'
+
+// Like https://github.com/brunobertolini/styled-by
+const styledBy = (property, mapping) => props => mapping[props[property]]
```

Comment:
This is handy 👍

Reply:
Now I think of it, I've used it in a few components. Should extract it somewhere like `utils`.

Reply:
Done.